Libplanet/Blockchain/BlockChain.States.cs

```diff
@@ -27,33 +26,16 @@ public IWorldState GetWorldState(HashDigest<SHA256>? stateRootHash)
         /// Gets the next world state in the <see cref="BlockChain"/>.
         /// </summary>
         /// <returns>The next world state.  If it does not exist, returns null.</returns>
-        public IWorldState? GetNextWorldState() => GetNextWorldState(Tip.Hash);
-
-        /// <summary>
-        /// Returns the next <see cref="IWorldState"/> in the BlockChain
-        /// at <paramref name="offset"/>.
-        /// </summary>
-        /// <param name="offset">The <see cref="BlockHash"/> of the <see cref="Block"/> to create
-        /// for which to create an <see cref="IWorldState"/>.</param>
-        /// <returns>
-        /// The next <see cref="IWorldState"/> at <paramref name="offset"/>.
-        /// Returns <see langword="null"/> if next state root hash does not exists.
-        /// </returns>
-        /// <exception cref="ArgumentException">Thrown when next state root hash exists,
-        /// but corresponding state root is not found in the <see cref="IStateStore"/>.
-        /// </exception>
-        /// <seealso cref="IWorldState"/>
-        public IWorldState? GetNextWorldState(BlockHash offset)
+        public IWorldState? GetNextWorldState()
         {
-            var nextSrh = Store.GetNextStateRootHash(offset);
-            if (nextSrh is { } srh)
+            if (GetNextStateRootHash() is { } srh)
```

Comment:
trivial, but variable name nsrh would be better for here.
```suggestion
            if (GetNextStateRootHash() is { } nsrh)
```